website/markdown/docs/examples/command-line-tools.mdx

```diff
@@ -0,0 +1,28 @@
+---
+name: Command Line Tools
+excerpt: 'This page documents how to declare and use Command Line Tool targets.'
+---
+
+import Message from '../components/message'
+
+# Command Line Tools
+
+On macOS we can build various command line tools (like tuist) this way you can define your project.
+
+```swift
+let project = Project(
+    name: "CLITool",
+    targets: [
+        Target(
+            name: "CLITool",
+            platform: .macOS,
+            product: .commandLineTool,
+            bundleId: "io.tuist.App",
+            infoPlist: .default,
+            sources: ["Sources/**"],
+        ),
```

Comment:
I'd extend it to have a library as a dependency. That way users will see that it's possible to define dependencies for CLI too.

Reply:
✅